pyqtgraph/graphicsItems/TextItem.py

```diff
@@ -114,6 +115,16 @@ def setAngle(self, angle):
     def setAnchor(self, anchor):
         self.anchor = Point(anchor)
         self.updateTextPos()
+
+    def setColor(self, color):
+        """
+        Set the color for this text.
+        
+        See QtGui.QGraphicsItem.setDefaultTextColor().
+        """
+        self.color = fn.mkColor(color)
+        self.textItem.setDefaultTextColor(self.color)
+        self.updateTextPos()
```

Comment:
No. It was just a mistake. I copy/paste setText method to create the setColor method and did not checkd carefully enough. Thanks!